recipes/mleap/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "mleap" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb384a239747f319297a0b4f2bb5c076f01cfd942d7f72a173494606f402fdad
+
+build:
+  number: 0
```

I think this can be `noarch`
```suggestion
  number: 0
  noarch: python
```